SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -89,17 +89,37 @@ public static void Main(string[] args)
             // Locate the Unreal Engine.
             Console.WriteLine("Finding Unreal Engine build.");
             string uproject = File.ReadAllText(projectFile, Encoding.UTF8);
-            string engineAssociationRegex = "..{8}-.{4}-.{4}-.{4}-.{12}.";
-            Match engineAssociationMatch = Regex.Match(uproject, engineAssociationRegex);
 
             string unrealEngine = "";
+            string engineAssociation = "";
```

Comment:
Might as well assign this directly to the projectJson version.

Reply:
Done